src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -11,31 +11,45 @@
+register_cli_argument('acr', 'sku', help='The SKU of the container registry', choices=['Basic', 'Managed_Basic', 'Managed_Standard', 'Managed_Premium'])
 register_cli_argument('acr', 'password_name', help='The name of password to regenerate', choices=['password', 'password2'])
 
 register_cli_argument('acr', 'resource_group_name', resource_group_name_type)
 register_cli_argument('acr', 'location', location_type)
 register_cli_argument('acr', 'tags', tags_type)
-register_cli_argument('acr create', 'admin_enabled', nargs='?', required=False, const='true', default=None, help='Indicates whether the admin user is enabled', choices=['true', 'false'])
-register_cli_argument('acr update', 'admin_enabled', nargs=1, help='Indicates whether the admin user is enabled', choices=['true', 'false'])
+register_cli_argument('acr create', 'admin_enabled', nargs='?', required=False, const='true', default='false', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
+register_cli_argument('acr update', 'admin_enabled', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
 
 register_cli_argument('acr', 'username', options_list=('--username', '-u'), help='The username used to log into a container registry')
 register_cli_argument('acr', 'password', options_list=('--password', '-p'), help='The password used to log into a container registry')
 
+register_cli_argument('acr repository delete', 'manifest', nargs='?', required=False, const='', default=None, help='The sha256 based digest of manifest to delete')
+register_cli_argument('acr repository delete', 'yes', options_list=('--yes', '-y'), action='store_true', help='Do not prompt for confirmation')
+
 register_cli_argument('acr create', 'registry_name', completer=None, validator=validate_registry_name)
 register_cli_argument('acr create', 'deployment_name', deployment_name_type, validator=None)
+register_cli_argument('acr create', 'location', location_type, validator=get_default_location_from_resource_group)
 register_cli_argument('acr check-name', 'registry_name', completer=None)
-register_cli_argument('acr create', 'storage_account_name', help='Default: A new storage account will be created. Provide the name of an existing storage account if you\'re recreating a container registry over a previous registry created storage account.', completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
-register_cli_argument('acr update', 'storage_account_name', help='Provide the name of an existing storage account if you\'re recreating a container registry over a previous registry created storage account.', completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
+
+register_cli_argument('acr webhook', 'registry_name', options_list=('--registry', '-r'), help='The name of the container registry', completer=get_resource_name_completion_list(ACR_RESOURCE_TYPE))
```

Comment:
Does this arg pick up the default registry name if it's configured? The help text below doesn't seem to indicate it does, and it would be ideal to have it be defaulted when the ACR default is set.